microsoft-edge/devtools-guide-chromium/autofill/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Autofill: Inspect and debug saved addresses"
+description: Inspect and debug address info that's been saved to Microsoft Edge.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 02/28/2025
+---
+<!-- Copyright Sofia Emelianova
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.  -->
+# Autofill: Inspect and debug saved addresses
+<!-- https://developer.chrome.com/docs/devtools/autofill -->
+
+Use the **Autofill** tool to inspect and debug address info that's been saved to Microsoft Edge.
+
+The **Autofill** tool records various events that the browser triggers when a user is filling in a form by using the browser's autofill feature -- for example, when the browser remembers your address and autofills it in an address field.
+
+![The Autofill tool](./index-images/autofill-tool.png)
+
+
+<!-- ====================================================================== -->
+## Overview
+
+The **Autofill** tool provides a convenient way to automatically fill forms on websites with saved addresses.  The **Autofill** tool in DevTools lets you inspect the mapping between your form fields, predicted autofill values, and saved data.
```

Comment:
```suggestion
The browser autofill feature provides a convenient way to automatically fill forms on websites with saved addresses. The **Autofill** tool in DevTools lets you inspect the mapping between your form fields, predicted autofill values, and saved data.
```
Maybe link to https://support.microsoft.com/en-us/microsoft-edge/automatically-fill-info-in-microsoft-edge-81da697c-9910-d9b8-d50a-1712d96f3db8 to explain what the browser autofill feature is.


moved PB [Comment from 3378](https://github.com/MicrosoftDocs/edge-developer/pull/3378/files/88196323953a6046e0faf6f2bfd45ef5f3de0331#r1975694937) to here.